src/wpanctl/tool-cmd-commissioner.c

```diff
@@ -305,7 +305,7 @@ int tool_cmd_commissioner(int argc, char* argv[])
 					goto bail;
 				}
 				if (strnlen(ext_addr, EXT_ADDRESS_LENGTH_CHAR+1) != EXT_ADDRESS_LENGTH_CHAR) {
-					fprintf(stderr, "%s: error: Wrong address length.%d \n", argv[0], strnlen(ext_addr, EXT_ADDRESS_LENGTH_CHAR+1));
+					fprintf(stderr, "%s: error: Wrong address length.%zu \n", argv[0], strnlen(ext_addr, EXT_ADDRESS_LENGTH_CHAR+1));
```

Comment:
Is `z` modifier designed to solve this portability issue? If it's not allowed, shall we make it clear in `CONTRIBUTING.md` or other place that we should avoid new features introduced in C99?
I think other contributor may also confused about what to use and what not.